Yosemite/YosemiteTests/Stores/ProductStoreTests.swift

```diff
@@ -1578,6 +1592,16 @@ private extension ProductStoreTests {
         return [defaultAttribute1]
     }
 
+    func sampleDownloadsMutated() -> [Networking.ProductDownload] {
```

Comment:
I suggest you to move it after `sampleDownloads()` 👍

Reply:
Moved